cmd/lakectl/cmd/common_helpers.go

```diff
@@ -201,14 +201,14 @@ func RetrieveError(response interface{}, err error) error {
 	return helpers.ResponseAsError(response)
 }
 
-func DieOnResponseError(response interface{}, err error) {
+func dieOnResponseError(response interface{}, err error) {
```

Comment:
It might be a handy function but as I see it, no method is using it expect for `DieOnErrorOrUnexpectedStatusCode`, so what are the odds the a new one will?
On top of that, developers that used to use `DieOnResponseError` might use it again as a force of habit which we want to prevent.
wdyt?

Reply:
Agree with @Jonathan-Rosenberg about this. we want to prevent users from using `DieOnResponseError`